docs/resources/Channel.md

```diff
@@ -843,6 +844,17 @@ user 125 in the content.
 }
 ```
 
+### Role Subscription Data Object
+
+###### Role Subscription Data Object Structure
+
+| Field                        | Type      | Description                                                           |
+| ---------------------------- | --------- | --------------------------------------------------------------------- |
+| role_subscription_listing_id | snowflake | the id of the sku and listing that the user is subscribed to          |
+| tier_name                    | string    | the name of the tier that the user is subscribed to                   |
+| total_months_subscribed      | integer   | the number of months that the user has been subscribed for            |
```

Comment:
If you want to be extra explicit, I would say `cumulative number of months` (vs consecutive)